trade-finance-manager-ui/server/controllers/case/index.js

```diff
@@ -449,7 +449,7 @@ const confirmTfmFacility = async (req, res) => {
       api
         .getParty(urn, userToken)
         // Non-existent party urn
-        .then((company) => (!company || !company.data ? Promise.resolve(false) : Promise.resolve(true))));
+        .then((company) => (!company?.data || company?.status !== 200 ? Promise.reject(false) : Promise.resolve(true))));
```

Comment:
Promise.reject() cannot be false have to be an error.
Does the calling function rely on `false` response?

Reply:
Yep there is a boolean check later on, thought I might be able to pass in false that would get wrapped by an error but you're right - can I leave as resolve(false) for now? This is used less as a rejected error and more as a if the company does or doesn't exist right nowu